zeroex/order_js.go

```diff
@@ -55,3 +56,49 @@ func (s SignedOrder) JSValue() js.Value {
 		"signature":             signature,
 	})
 }
+
+func (c ContractEvent) JSValue() js.Value {
+	m := map[string]interface{}{
+		"blockHash": c.BlockHash.Hex(),
+		"txHash":    c.TxHash.Hex(),
+		"txIndex":   c.TxIndex,
+		"logIndex":  c.LogIndex,
+		"isRemoved": c.IsRemoved,
+		"kind":      c.Kind,
+	}
+	switch c.Kind {
+	case "ERC20TransferEvent":
```

Comment:
You also might not need this switch statement, but I'm less sure about whether `js.ValueOf` recursively iterates through each key/value pair. At the very least you should be able to cast `c.Parameters` to `js.Wrapper` instead.

Reply:
Yep, was able to remove this one as well.